src/semantic/color.tokens.json5

```diff
@@ -334,6 +334,64 @@
           },
         },
       },
+      "on-additional": {
```

Comment:
imho we should try and find a different name for this one 🤓

Reply:
Any idea?

Reply:
will think about it!